src/main/scala/edu/ie3/powerFactory2psdm/config/ConversionConfig.scala

```diff
@@ -6,52 +6,146 @@
 
 package edu.ie3.powerFactory2psdm.config
 
-import edu.ie3.powerFactory2psdm.config.ConversionConfig.ModelConfigs
+import edu.ie3.powerFactory2psdm.config.ConversionConfig.StatGenModelConfigs
 
-final case class ConversionConfig(modelConfigs: ModelConfigs)
+final case class ConversionConfig(modelConfigs: StatGenModelConfigs)
 
+/**
+  * Config used for the grid conversion
+  */
 object ConversionConfig {
+  val loadFlowSource: String = "load flow"
+  val basicDataSource: String = "basic data"
```

Comment:
That is true! Since I can not "read in" enumerations from the config file without adding another Enumeration Framework (https://github.com/pureconfig/pureconfig/tree/master/modules/enumeratum) I would just go with a sealed trait `ParameterSource` and Final case objects of that trait (e.g. `LoadFlowSource`)